test/SlowTests/RavenDB-21679.cs

```diff
@@ -0,0 +1,187 @@
+                };
+
+                await ExpirationHelper.SetupExpiration(store, Server.ServerStore, expirationConfiguration);
+
+                var backupConfiguration = Backup.CreateBackupConfiguration(backupPath, fullBackupFrequency: "* */1 * * *", incrementalBackupFrequency: "* */2 * * *");
+                await store.Maintenance.SendAsync(new UpdatePeriodicBackupOperation(backupConfiguration));
+
+                store.Maintenance.Send(new PutClientConfigurationOperation(new ClientConfiguration { ReadBalanceBehavior = ReadBalanceBehavior.None, LoadBalanceBehavior = LoadBalanceBehavior.None, LoadBalancerContextSeed = 0, Disabled = false }));
+
+                await RevisionsHelper.SetupRevisionsAsync(store);
+
+                DatabaseRecord record;
+                using (Server.ServerStore.ContextPool.AllocateOperationContext(out TransactionOperationContext context))
+                using (context.OpenReadTransaction())
+                {
+                    record = Server.ServerStore.Cluster.ReadDatabase(context, store.Database);
+                }
+
+                Assert.True(record.RavenConnectionStrings.ContainsKey("RavenConnectionString"));
+                Assert.NotNull(record.Expiration);
+                Assert.True(record.PeriodicBackups.Count > 0);
+                Assert.NotNull(record.Client);
+                Assert.NotNull(record.Revisions);
+
+                var operation = await store.Smuggler.ExportAsync(new DatabaseSmugglerExportOptions(), file);
+                await operation.WaitForCompletionAsync(TimeSpan.FromMinutes(1));
+            }
+
+            using (var store = GetDocumentStore(new Options()
+            {
+                AdminCertificate = adminCert,
+                ClientCertificate = userCert,
+                ModifyDatabaseName = _ => dbName2
+            }))
+            {
+                WaitForUserToContinueTheTest(store, clientCert:userCert);
+                var operation = await store.Smuggler.ImportAsync(new DatabaseSmugglerImportOptions(), file);
+                await operation.WaitForCompletionAsync(TimeSpan.FromMinutes(1));
+
+                var output = await store
+                    .Maintenance
+                    .SendAsync(new GetIndexOperation(input.Name));
+
+                Assert.Null(output);
+
+                DatabaseRecord record;
+                using (Server.ServerStore.ContextPool.AllocateOperationContext(out TransactionOperationContext context))
+                using (context.OpenReadTransaction())
+                {
+                    record = Server.ServerStore.Cluster.ReadDatabase(context, store.Database);
+                }
+
+                Assert.False(record.RavenConnectionStrings.ContainsKey("RavenConnectionString"));
+                Assert.Null(record.Expiration);
+                Assert.False(record.PeriodicBackups.Count > 0);
+                Assert.Null(record.Client);
+                Assert.Null(record.Revisions);
```

Comment:
Can we also verify that the relevant warnings / errors have been written to the import log?

Reply:
Done